### PR TITLE
[WIP] Docs refresh

### PR DESCRIPTION
**Pull request by bryevdv**
This PR does a few things

* fixes a bug with enabling coverage reporting
* makes a dedicated user guide chapter
* adds some draft example notebooks
* makes a dedicated dev guide chapter

Help wanted: 

* what sections of the user guide are missing or need to be changed?
* notebooks need to be fleshed out with more information about CLLR
* what is most important to cover in (driver) configuration options section?
* a benchmarking chapter? could go in either user's guide or dev guide

NOTE: to build locally you will need new dependencies listed in https://github.com/nv-legate/legate.core/pull/579

---

## Front page

image

## Usage chapter

image

## Settings

image

## Notebook

image

### CODE REVIEW

File: cunumeric/runtime.py
Comment (bryevdv) [inline cunumeric/runtime.py:173]: `args` is gone now so this `hasattr` always failed and so coverage reporting would never be activated.

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/black_scholes.ipynb:105]: typo: implement

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/black_scholes.ipynb:139]: should this be "one million put and call options"?

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/black_scholes.ipynb:161]: These examples are usually much more compelling if you can compare CPU vs. multi-GPU timings.  If we can get the resource-scoping-pr merged, we could compute this twice with a different machine and show GPU times being way smaller.

File: docs/cunumeric/source/user/notebooks/cholesky.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/cholesky.ipynb:144]: A better example might be a diagonally dominant banded matrix? which will be a little more representative of something you would have to use in a PDE solver.

N = 1000
off_diag = np.ones(N-1)
A = 3 * np.eye(N) + np.diag(off_diag, -1) + np.diag(off_diag, 1)

File: docs/cunumeric/source/user/notebooks/cholesky.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/cholesky.ipynb:160]: Same comment as black_scholes. The examples are a lot more interesting if you have CPU and multi-GPU timings in the same notebook.

File: docs/cunumeric/source/user/notebooks/cholesky.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/cholesky.ipynb:188]: There are two empty cells at the bottom, which looks a little untidy.

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/stencil.ipynb:60]: timing should probably be explained as in the other notebooks. We may actually want to mention about the asynchronous nature of the computation and why regular python timing will not work. Actually, that might be a great 4th notebook.  Show python time giving 0ms because everything is deferred and legate timing giving you real times.

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/stencil.ipynb:140]: semicolon

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/stencil.ipynb:138]: the 100x100 plot had too much green and it was hard to distinguish the boundary. Maybe smaller?

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/stencil.ipynb:180]: semicolon

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [inline docs/cunumeric/source/user/notebooks/stencil.ipynb:77]: Same comment as before. Timings would be a lot more interesting if shown for both CPU and multi-GPU

**Review by jjwilke — CHANGES_REQUESTED** [review]
Super picky comments. Overall this looks great.

I'm currently unable to build the documentation locally - blocked on:
```
sphinx.errors.SphinxWarning: /Users/jwilke/miniconda3/envs/cunumeric-docs/lib/python3.9/site-packages/cunumeric/random/__init__.py:docstring of cunumeric.random.binomial:54:Footnote [1] is not referenced.
```

Didn't have time to dive further into it.

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst:10]: I'm happy to change the verbiage but this came from the existing README so we should get agreement on changing things in both places cc @manopapad

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb:161]: That's not going to be possible in a single notebook, is it? AFAIK the Realm runtime has to be configured at startup and can't be changed. I'm also not sure we can assume users who want to run the notebooks will have multi-gpus. cc @manopapad 

I do think comparisons would be great to have, but I think we should do them, and write up the results with some plots, in a ReST chapter.

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:140]: That was intentional to suppress the annoying plot object string repr

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:180]: intentional, as above

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb:161]: I do think we could include information in the notebooks about how to re-run them with a different configuration (but I do think they would have to be exited, and re-run)

File: BUILD.md
Comment (bryevdv) [reply BUILD.md:40]: I would also improve the excessive hyphenation 

> cuNumeric uses the same build workflow as Legate Core, based on [CMake]() and [scikit-build]().

Comment (bryevdv) [reply BUILD.md:40]: @manopapad should it be "Legate Core" in these contexts? Or just "Legate" ?

**Comment by bryevdv** [comment]
@ipdemes 100% I was planning on including that in the TBD chapter on `legate` configuration. Do you have any specific suggestions for things or use cases to emphasize?

File: docs/cunumeric/source/user/notebooks/cholesky.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/cholesky.ipynb:144]: I was thinking the same thing, will change here but FWIW this is more or less just the existing `cholesky.py` example, so not sure if we'd want to make that more realistic/interesting too.

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:60]: A notebook for timing seems good, do we also want to have a sentence or two in all the notebooks for reinforcement? If so, can you suggest some verbiage?

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:138]: smaller grid size? 100x100 already seems unimpressively small TBH, maybe we could just omit the first plot (or even both plots... I like to see visual things but if they are not actually useful they can be removed)

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (jjwilke) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb:161]: With the resource-scoping-pr, we could do the following:

Cell #1
```
def function_that_does_everything(...):

```

Cell #2:
```
with machine.only("cpu"):
  function_that_does_everything()
```

Cell #3:
```
with machine.only("gpu"):
  function_that_does_everything()
```

Users without a GPU would just not run cell #3. Alternatively, cell #3 should fail gracefully without completely crashing the kernel if there are no GPUs. This might make more sense for a follow-up PR. Seeing GPU times that are shorter than CPU times definitely hit the reward centers of user's brains and get people engaged.

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:60]: Maybe...


Because of Legate's deferred execution model, `legate.timing` should be used instead of standard Python `datetime` utilities. Python `datetime.now` would return the time a task is *enqueued*, not necessarily the time a task finishes executing.

@manopapad @magnatelee

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:140]: oh, huh, okay. Haven't seen that before. 👍

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (jjwilke) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:138]: Yeah, I was thinking to show a 10x10 plot and then run the 100x100 example. On second look, I think the plot is fine. Maybe just more clarification in the text.

"As we can see in the plot, the initial conditions have an absolute zero boundary top, left, and right (dark blue) with a higher boundary value on the bottom (yellow) with an intermediate value (green) in the interior."

"After running, we can see the boundary conditions are preserved, but the interior values have adjusted to average values in-between the boundaries."

Not sure what the standard is, but the input values were a bit weird to me. Instead of specifying the absolute zero temperature boundaries as 0, they are given as -273.15.

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb:161]: > With the resource-scoping-pr, 

What PR are you referring to? Is it something already merged, or soon to be?

> Seeing GPU times that are shorter than CPU times definitely hit the reward centers of user's brains and get people engaged.

Of course, but we need to make sure these examples actually demonstrate that. The GPU tests often run more slowly for me locally, presumably because they are small and some startup time dominates.

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb]: https://github.com/nv-legate/cunumeric/pull/805/commits/971855c33f59ee06de0f249dd4a9640685234025

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb]: https://github.com/nv-legate/cunumeric/pull/805/commits/971855c33f59ee06de0f249dd4a9640685234025

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:151]: Also I guess FWIW the boundary does seem noticeable to me as it is 

image

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb]: https://github.com/nv-legate/cunumeric/pull/805/commits/940fe8ad19b139fab627f6db0c80df61d7a90eeb

File: docs/cunumeric/source/user/notebooks/cholesky.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/cholesky.ipynb:193]: https://github.com/nv-legate/cunumeric/pull/805/commits/940fe8ad19b139fab627f6db0c80df61d7a90eeb

**Comment by bryevdv** [comment]
@jjwilke I am not sure about your build issue. Things are building locally for me and more strange. AFAICT there is not any [footnote](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#footnotes) anywhere in the docstring of the referenced function.

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst:10]: Let's save tone/verbiage updates for a dedicated and comprehensive review across all docs, examples, etc.

File: docs/cunumeric/source/user/notebooks/black_scholes.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/black_scholes.ipynb:174]: This will go in a follow-up PR once resource scoping is merged

File: docs/cunumeric/source/user/notebooks/stencil.ipynb
Comment (bryevdv) [reply docs/cunumeric/source/user/notebooks/stencil.ipynb:151]: @manopapad any preferences here?

**Review by jjwilke — APPROVED** [review]
LGTM. 

Follow-up PRs seem like the right way to handle things.

**Comment by bryevdv** [comment]
>  do you think it would be a good idea to to add something like "WARNING" section on the "usage" page saying for which applications cuNumeric is a good option and for which it can be slower than NumPy ?

@ipdemes I think that would be reasonable, but I don't actually know what that boundary is :) Is there some existing text I can adapt? If not can you make an issue with a rough outline of what you think needs to be said, and it can go in a follow-up PR

File: docs/cunumeric/source/index.rst
Comment (manopapad) [inline docs/cunumeric/source/index.rst:35]: This link sounds wrong

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [inline docs/cunumeric/source/user/usage.rst:41]: Note here that LEGATE_CONFIG has lower precedence.

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [inline docs/cunumeric/source/user/usage.rst:59]: Is this section empty by design?

File: docs/cunumeric/source/developer/CONTRIBUTING.md
Comment (manopapad) [inline docs/cunumeric/source/developer/CONTRIBUTING.md:1]: Can this file be linked to the top-level CONTRIBUTING.md, like BUILD.md is?

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [reply docs/cunumeric/source/user/usage.rst:41]: Or maybe note that in configuration.rst, and link that page from here ("see configuration.rst for more details on configuration options").

**Comment by bryevdv** [comment]
OK there is a first cut for multi-node execution derived largely from the legate README

File: docs/cunumeric/source/developer/CONTRIBUTING.md
Comment (bryevdv) [reply docs/cunumeric/source/developer/CONTRIBUTING.md]: weird I thought I had https://github.com/nv-legate/cunumeric/pull/805/commits/7bb0815e124eef9c4e58ea86cf0d3edc5ce8a5f3

File: BUILD.md
Comment (manopapad) [reply BUILD.md:40]: Technically speaking Legate refers to the whole ecosystem, and Legate Core is the core library that every other library sits on top of. But we've been pretty loose about this, and may be renaming the components soon anyway, so I wouldn't worry about it too much right now.

File: docs/cunumeric/source/user/usage.rst
Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst]: I moved config to an entire separate chapter, this heading was removed in https://github.com/nv-legate/cunumeric/pull/805/commits/4f1b1f971576178da50a99ef5c28f2711d64a6b2

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst:35]: In what way? I took it from the readme and it seems to go to the right notebook

File: docs/cunumeric/source/user/usage.rst
Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst:41]: > Note here that LEGATE_CONFIG has lower precedence.

just checking,  you mean lower than command line options?

File: docs/cunumeric/source/developer/testing.rst
Comment (manopapad) [inline docs/cunumeric/source/developer/testing.rst:14]: sentence ends midway through?

File: docs/cunumeric/source/developer/testing.rst
Comment (manopapad) [inline docs/cunumeric/source/developer/testing.rst:103]:
```suggestion
  Number of OpenMP processors per node to use
```

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst:35]: I agree but that is what it is in the README where I copied from w/o paying close attention apparently. What is the best product link to use?

Comment (bryevdv) [reply docs/cunumeric/source/index.rst:35]: https://www.nvidia.com/en-us/data-center/dgx-superpod/ ?

File: docs/cunumeric/source/developer/testing.rst
Comment (manopapad) [inline docs/cunumeric/source/developer/testing.rst:109]:
```suggestion
  Number of utility CPUs to reserve for runtime services
```

File: docs/cunumeric/source/developer/testing.rst
Comment (bryevdv) [reply docs/cunumeric/source/developer/testing.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/81750a565fb90bda9c79ce2aecc070bcd3837ab5

File: docs/cunumeric/source/developer/testing.rst
Comment (bryevdv) [reply docs/cunumeric/source/developer/testing.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/81750a565fb90bda9c79ce2aecc070bcd3837ab5

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/b393a112496883d47dba96cf786fd10c8ace63b1

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [inline docs/cunumeric/source/user/usage.rst:68]: I don't think we force "mpirun" anymore, regardless if --nodes is present:

```
~/cunumeric> legate --nodes 1 --verbose a.py

--- Legion Python Configuration ------------------------------------------------

Legate paths:
  legate_dir       : /Users/mpapadakis/legate.core
  legate_build_dir : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build
  bind_sh_path     : /Users/mpapadakis/legate.core/bind.sh
  legate_lib_path  : /Users/mpapadakis/legate.core/build/lib

Legion paths:
  legion_bin_path       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/bin
  legion_lib_path       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/lib
  realm_defines_h       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/runtime/realm_defines.h
  legion_defines_h      : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/runtime/legion_defines.h
  legion_spy_py         : /Users/mpapadakis/legion/tools/legion_spy.py
  legion_prof_py        : /Users/mpapadakis/legion/tools/legion_prof.py
  legion_python         : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/bin/legion_python
  legion_module         : /Users/mpapadakis/legion/bindings/python/build/lib
  legion_jupyter_module : /Users/mpapadakis/legion/jupyter_notebook

Versions:
  legate_version : 23.03.00.dev+36.g7e883c5

Command:
  /Users/mpapadakis/legate.core/bind.sh --launcher local -- /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/bin/legion_python -ll:py 1 -lg:local 0 -ll:cpu 4 -ll:util 2 -ll:csize 4000 -ll:networks none -level openmp=5 -lg:eager_alloc_percentage 50 -ucx:tls_host '^dc,ud' a.py

Customized Environment:
  DYLD_LIBRARY_PATH=/Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/lib:/Users/mpapadakis/legate.core/build/lib
  GASNET_MPI_THREAD=MPI_THREAD_MULTIPLE
  LEGATE_MAX_DIM=4
  LEGATE_MAX_FIELDS=256
  NCCL_LAUNCH_MODE=PARALLEL
  PYTHONDONTWRITEBYTECODE=1
  PYTHONPATH=/Users/mpapadakis/legion/bindings/python/build/lib:/Users/mpapadakis/legion/jupyter_notebook:/Users/mpapadakis/legate.core/legate
  REALM_BACKTRACE=1
  REALM_UCP_BOOTSTRAP_PLUGIN=/Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/lib/realm_ucp_bootstrap_mpi.so
  UCX_CUDA_COPY_MAX_REG_RATIO=1.0
  UCX_IB_RCACHE_PURGE_ON_FORK=n
  UCX_MULTI_LANE_MAX_RATIO=1.0
  UCX_RC_TX_POLL_ALWAYS=y

--------------------------------------------------------------------------------

[0 - 10bb70dc0]    0.000271 {4}{threads}: reservation ('CPU proc 1d00000000000003') cannot be satisfied

~/cunumeric> legate --nodes 2 a.py
ERROR: Could not initialize driver, path config and exception follow:

--- Legion Python Configuration ------------------------------------------------

Legate paths:
  legate_dir       : /Users/mpapadakis/legate.core
  legate_build_dir : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build
  bind_sh_path     : /Users/mpapadakis/legate.core/bind.sh
  legate_lib_path  : /Users/mpapadakis/legate.core/build/lib

Legion paths:
  legion_bin_path       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/bin
  legion_lib_path       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/lib
  realm_defines_h       : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/runtime/realm_defines.h
  legion_defines_h      : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/runtime/legion_defines.h
  legion_spy_py         : /Users/mpapadakis/legion/tools/legion_spy.py
  legion_prof_py        : /Users/mpapadakis/legion/tools/legion_prof.py
  legion_python         : /Users/mpapadakis/legate.core/_skbuild/macosx--x86_64-3.9/cmake-build/_deps/legion-build/bin/legion_python
  legion_module         : /Users/mpapadakis/legion/bindings/python/build/lib
  legion_jupyter_module : /Users/mpapadakis/legion/jupyter_notebook

Versions:
  legate_version : 23.03.00.dev+36.g7e883c5

--------------------------------------------------------------------------------

Traceback (most recent call last):
  File "/Users/mpapadakis/opt/miniconda3/envs/legate/bin/legate", line 33, in <module>
    sys.exit(load_entry_point('legate.core', 'console_scripts', 'legate')())
  File "/Users/mpapadakis/legate.core/legate/driver/__init__.py", line 37, in main
    return _main(argv)
  File "/Users/mpapadakis/legate.core/legate/driver/main.py", line 73, in legate_main
    driver = prepare_driver(argv, LegateDriver)
  File "/Users/mpapadakis/legate.core/legate/driver/main.py", line 54, in prepare_driver
    raise e
  File "/Users/mpapadakis/legate.core/legate/driver/main.py", line 49, in prepare_driver
    driver = driver_type(config, system)
  File "/Users/mpapadakis/legate.core/legate/driver/driver.py", line 68, in __init__
    self.launcher = Launcher.create(config, system)
  File "/Users/mpapadakis/legate.core/legate/driver/launcher.py", line 118, in create
    return SimpleLauncher(config, system)
  File "/Users/mpapadakis/legate.core/legate/driver/launcher.py", line 295, in __init__
    raise RuntimeError(RANK_ERR_MSG)
RuntimeError: Could not detect rank ID on multi-rank run with no --launcher provided. If you
want Legate to use a launcher, e.g. mpirun, internally (recommended), then you
need to specify which one to use by passing --launcher. Otherwise you need to
invoke the legate script itself through a launcher.
```

Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst:68]: Legate readme is out of date then

Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst:68]: Can you suggest some verbiage and I will update in both places

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [inline docs/cunumeric/source/user/usage.rst:73]:
```suggestion
Cunumeric programs can be run in parallel by using the ``--nodes`` option to
the ``legate`` driver, followed by the number of nodes to be used.
When running on 2+ nodes, a task launcher must be specified.
```

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [inline docs/cunumeric/source/user/usage.rst:96]:
```suggestion
The ``lgpatch`` script (in the same location as the ``legate`` executable) can
```

File: docs/cunumeric/source/index.rst
Comment (manopapad) [reply docs/cunumeric/source/index.rst]: LOL. Yes, that's a good link. Do you mind updating the README as well? 🤦

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [reply docs/cunumeric/source/user/usage.rst:41]: Right

File: docs/cunumeric/source/user/usage.rst
Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0

File: docs/cunumeric/source/user/usage.rst
Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0

File: docs/cunumeric/source/developer/testing.rst
Comment (bryevdv) [reply docs/cunumeric/source/developer/testing.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0

File: docs/cunumeric/source/user/usage.rst
Comment (bryevdv) [reply docs/cunumeric/source/user/usage.rst:41]: https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst]: https://github.com/nv-legate/cunumeric/pull/805/commits/111bc9affb4f10755b446997fbd2126c738f2ce0 (both)

File: docs/cunumeric/source/user/usage.rst
Comment (manopapad) [reply docs/cunumeric/source/user/usage.rst]: Yup, this suggestion would work for the README as well I think https://github.com/nv-legate/cunumeric/pull/805#discussion_r1119401744

File: docs/cunumeric/source/index.rst
Comment (bryevdv) [reply docs/cunumeric/source/index.rst]: well I guess that's not the right commit link,  but both are updated.

**Review by manopapad — APPROVED** [review]

**Pull request merged**
